### PR TITLE
Bugfix/upload multiple pic

### PR DESCRIPTION
**Pull request by LordAldi**
improve from pr (#62)
implement multiple item upload, connect to backend API

### CODE REVIEW

File: src/pages/upload/UploadPage.js [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+import "./upload.less";
+import { useApi } from "../../hooks/useApi";
+import { navigate } from "@reach/router";
+import mocks from "../../mocks";
+const { Option } = Select;
+
+const UploadPage = () => {
+  const [form] = Form.useForm();
+  const { reqHeader } = useAuth();
+  console.log(reqHeader);
```
Comment (ariefrahmansyah) [inline src/pages/upload/UploadPage.js]: Please remove this logging

File: src/pages/upload/UploadPage.js [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+  useEffect(() => {
+    console.log("form", form);
+  }, [form]);
```
Comment (ariefrahmansyah) [inline src/pages/upload/UploadPage.js]: Remove this logging too

File: src/pages/upload/UploadPage.js [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    console.log("Failed:", errorInfo);
+  };
+
+  const onFieldsChange = (changedFields, allFields) => {
+    console.log(changedFields, allFields);
+  };
+
+  function handleChange(value) {
+    console.log(`selected ${value}`);
+  }
```
Comment (ariefrahmansyah) [inline src/pages/upload/UploadPage.js]: Remove these logging too

File: src/providers/auth/context.js [hunk 4]
```diff
@@ -14,7 +14,7 @@ const AuthContext = createContext(null);
 
 export function AuthProvider({ children }) {
   const [reqHeader, setReqHeader] = useState({
-    "Device-id": browserName,
+    "Device-id": browserName + 3234242,
```
Comment (ariefrahmansyah) [inline src/providers/auth/context.js]: What is this value?

**Review by ariefrahmansyah — CHANGES_REQUESTED** [review]

File: src/providers/auth/context.js (hunk 4 above)
Comment (LordAldi) [reply src/providers/auth/context.js]: oww sorry that will change when merge, for testing purpose only

**Comment by ariefrahmansyah** [comment]
I've tried submitting a product on Netlify deploy preview site, but it seems the upload still fails. Can you check again?

Would be nice if you share the video recording of upload feature and put it on PR description as well.

**Comment by LordAldi** [comment]
> I've tried submitting a product on Netlify deploy preview site, but it seems the upload still fails. Can you check again?
> 
> Would be nice if you share the video recording of upload feature and put it on PR description as well.

pic upload or product upload to backend, for backend we still waiting. but its matching with api doc should not be a problem.

**Comment by LordAldi** [comment]
> I've tried submitting a product on Netlify deploy preview site, but it seems the upload still fails. Can you check again?


https://user-images.githubusercontent.com/49033706/124809941-e7100600-df8a-11eb-882b-17d0c5f1d7e2.mp4
image

**Review by ariefrahmansyah — APPROVED** [review]

**Review by fadelmvi — APPROVED** [review]

**Pull request merged**